sdk/core/Azure.Core.TestFramework/src/RecordMatcher.cs

```diff
@@ -58,6 +60,16 @@ public RecordMatcher(bool compareBodies = true)
             "x-ms-correlation-request-id"
         };
 
+        /// <summary>
+        /// Query parameters whose values can change between recording and playback without causing URI matching
+        /// to fail. The presence or absence of the query parameter itself is still respected in matching.
+        /// </summary>
+        public HashSet<string> VolatileQueryParameters = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
Should be called IgnoredQueryParameters.

Reply:
Ignored isn't completely accurate, because it is the value that is ignored as opposed to the presence of the query parameter.

Reply:
It's still closer to the truth then volatile.

Reply:
I suppose its closer to the established convention, but Volatile seems closer to the truth of how this works. I will update this to match the convention.

Reply:
Should it be `ExcludeQueryParameters` to match `ExcludeHeaders`?